extensions/amp-analytics/0.1/vendors.js

```diff
@@ -198,6 +198,43 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'colanalytics': {
+    'vars': {
+      'id': '2964',
```

Comment:
Are these supposed to be specified by the publisher or do they stay same for eveyone? 

If they stay same for everyone, keep them here. If not, remove them from here. If you are around on slack, please ping me again and I can help you walk through it.